setup.py

```diff
@@ -45,8 +45,11 @@ def run_tests(self):
         'Topic :: Software Development :: Quality Assurance'
     ],
     entry_points={
-        'console_scripts': ['vulture = vulture:main'],
+        'console_scripts': ['vulture = vulture.core:main'],
     },
     tests_require=['pytest', 'pytest-cov'],
     cmdclass={'test': PyTest},
+    packages=['vulture'],
+    package_dir={'vulture': 'vulture'},
+    package_data={'vulture': ['whitelists/**/*.py', ]},
```

Comment:
To be clear: please add packages and package_dir in this PR and package_data in PR #50.